### ISSUE TITLE
Ember-Data 2.11.0 Infinit Ember.deprecate Warnings When Using this.store.pushPayload

### ISSUE THREAD

**Issue by tomoguisuru** [lytics/ember-data-model-fragments#234]
You probably are already aware of this issue at this point but, if you try to push a response into the store using `this.store.pushPayload(results)` and any of the models in the payload have fragments, you get a nasty error (see attached) repeated until your browser crashes.

Tested with a model that had a fragment that was just boolean properties, nothing fancy, and removing the fragment property allowed the model to be pushed using `pushPayload`

I also tested with Ember-Data 2.10.0 and was able to push with the fragment included

Ember 2.11.0
`this.store.pushPayload(results)`
With Fragment - failed with infinite Ember.deprecate message
Without Fragment - Successfully pushed records

User Model
```js
import MF from 'model-fragments';

const {
  Model,
  attr,
  hasMany,
  belongsTo
} = DS;

const {
  fragment
} = MF;

const UserModel = Model.extend({
  email:                      attr('string'),
  first_name:                 attr('string'),
  last_name:                  attr('string'),
  permissions:                fragment('permission'),
});

export default UserModel;
```

Permissions Fragment
```js
import DS from 'ember-data';
import MF from 'model-fragments';

const {
  attr
} = DS;

const {
  Fragment
} = MF;

const PermissionsFragment = Fragment.extend({
  can_view_user_management:   attr('boolean'),
  can_edit_users:             attr('boolean'),
  can_edit_groups:            attr('boolean'),
});

export default PermissionsFragment;
```

Sample payload push with error
```js
const results = {
  user: {
  email: "bob@happytrees.com",
  first_name: "Bob",
  last_name: "Ross",
   permissions: {
    can_view_user_management:   true,
    can_edit_users:             true,
    can_edit_groups:            true,
  }
};

this.store.pushPayload(results);
```
![screenshot 2017-01-12 11 24 15](https://cloud.githubusercontent.com/assets/569332/21908742/3d29928c-d8d2-11e6-9e8d-9bfd00dec68b.png)

**Comment by jakesjews** [comment]
We have a fix at https://github.com/lytics/ember-data-model-fragments/pull/227 we're just waiting on Ember 2.11 to release so it can be a clean upgrade

**Comment by jakesjews** [comment]
@tomoguisuru I'm going to close this as a duplicate of https://github.com/lytics/ember-data-model-fragments/issues/224
